src/components/LevelFilter.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import "../styles/LevelFilter.css";
+
+const LevelFilter = ({ levelsList, activeLevel, setActiveLevel }) => {
```

Comment:
Since this code is very similar to the subject filter, it could probably become a more general component.